org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporterTest.java

```diff
@@ -439,6 +441,41 @@ public void testJava14Project() throws Exception {
 		testJavaProjectWithPreviewFeatures("14", true /* The project has enabled preview features in the jdt setting*/, JavaCore.IGNORE);
 	}
 
+	@Test
+	public void testSubprojects() throws Exception {
```

Comment:
can you check that, when you set/change the target compilation in the root project, the children are updated?

Reply:
Done.